docs/_data/mesheryctlcommands/lifecycle/system/status/command/status.yaml

```diff
@@ -0,0 +1,4 @@
+name: status
```

Comment:
@umairnsr87 Could you please delete this file from the commit? As it is not relevant to the issue